src/importexport/musicxml/internal/musicxml/musicxmlsupport.h

```diff
@@ -236,9 +236,11 @@ extern void domError(const QDomElement&);
 extern void domNotImplemented(const QDomElement&);
 
 extern QString accSymId2MxmlString(const SymId id);
+extern QString accSymId2SmuflMxmlString(const SymId id);
 extern QString accidentalType2MxmlString(const AccidentalType type);
-extern AccidentalType mxmlString2accidentalType(const QString mxmlName);
-extern SymId mxmlString2accSymId(const QString mxmlName);
+extern QString accidentalType2SmuflMxmlString(const AccidentalType type);
+extern AccidentalType mxmlString2accidentalType(const QString mxmlName, const QString smufl);
+extern SymId mxmlString2accSymId(const QString mxmlName, const QString smufl = ""); //#
```

Comment:
What's that comment for?

Reply:
oh That was mark for me. I forgot to remove it :)

Reply:
Ping ;-)